src/cryptsetup/cryptsetup.c

```diff
@@ -173,29 +173,25 @@ static int parse_one_option(const char *option) {
 
                 r = safe_atou(val, &arg_sector_size);
                 if (r < 0) {
-                        log_error_errno(r, "Failed to parse %s, ignoring: %m", option);
+                        log_warning_errno(r, "Failed to parse %s, ignoring: %m", option);
                         return 0;
                 }
 
                 if (arg_sector_size % 2) {
-                        log_error("sector-size= not a multiple of 2, ignoring.");
+                        log_warning("sector-size= not a multiple of 2, ignoring.");
                         return 0;
                 }
 
-                if (arg_sector_size < CRYPT_SECTOR_SIZE || arg_sector_size > CRYPT_MAX_SECTOR_SIZE) {
-                        log_error("sector-size= is outside of %u and %u, ignoring.", CRYPT_SECTOR_SIZE, CRYPT_MAX_SECTOR_SIZE);
-                        return 0;
```

Comment:
some returns, but not others, are also being removed, was that intentional?

Reply:
As anyway the function will return with 0 soon.